### PR TITLE
Calculate picks

### PR DESCRIPTION
**Pull request by alexgaiv**

### CODE REVIEW

File: Image_processing/main.cpp [hunk 1]
```diff
@@ -13,31 +13,40 @@
...
 
-int calculateIntensity(cv::Vec3b &pixel);
-void showImage(cv::Mat* image);
+uchar calculateIntensity(const cv::Vec3b &pixel);
 
-class Histogram {
+class Histogram
+{
 public:
 	int hist[256];
```
Comment (PovelikinRostislav) [inline Image_processing/main.cpp]: Needed to add comments what are vecMax/Min to have a chance to revise after a long time

File: Image_processing/main.cpp [hunk 2]
```diff
@@ -13,31 +13,40 @@
...
-		for (int i = 0; i < image->cols; i++) {
-			for (int j = 0; j < image->rows; j++) {
-				cv::Vec3b pixColor = image->at<cv::Vec3b>(cv::Point(i, j));
-				int intensity = calculateIntensity(pixColor);
+	struct Pick
+	{
+		int leftBound;
+		int rightBound;
+		int extremum;
+		int area;
```
Comment (PovelikinRostislav) [inline Image_processing/main.cpp]: Pls add comment what is area?

File: Image_processing/main.cpp [hunk 3]
```diff
@@ -13,31 +13,40 @@
 -cleaning peaks under threshold
 */
 
+using namespace cv;
+using namespace std;
 
-int calculateIntensity(cv::Vec3b &pixel);
-void showImage(cv::Mat* image);
+uchar calculateIntensity(const cv::Vec3b &pixel);
```
Comment (PovelikinRostislav) [inline Image_processing/main.cpp]: It's good to explain why 'uchar': I mean comments in code that this is 0-255

File: Image_processing/main.cpp [hunk 4]
```diff
@@ -46,18 +55,41 @@ class Histogram {
 		for (int i = 0; i < histSize - 1; i++){
 			line(histImage, cv::Point(i, hist_h), cv::Point(i, hist_h - hist[i] / histSize), cv::Scalar(255, 255, 255), 1, 8, 0);
```
Comment (PovelikinRostislav) [inline Image_processing/main.cpp]: Again: comments about 'line' function 
Brief topic of that func

File: Image_processing/main.cpp [hunk 5]
```diff
@@ -46,18 +55,41 @@ class Histogram {
...
-		showImage(&histImage);
+		imshow(windowName, histImage);
 	}
 
-	void searchLocalMax() {
-		if (hist[0] > hist[1])
-			vecMax.push_back(0);
+	void searchLocalMax()
+	{
+		// нужно избавиться от нулевых значений по краям гистграммы
```
Comment (PovelikinRostislav) [inline Image_processing/main.cpp]: it's better to use ENG comments if we don't want to have any problems with different OS or configurations. 
This is not OK manner, I think

File: Image_processing/main.cpp (hunk 2 above)
Comment (alexgaiv) [inline Image_processing/main.cpp]: area is area under the pick

Comment (PovelikinRostislav) [inline Image_processing/main.cpp]: square?

**Pull request closed without merging**